aten/src/ATen/record_function.cpp

```diff
@@ -452,6 +460,8 @@ void RecordFunction::before(
   state_->sequence_nr_ = sequence_nr;
   state_->thread_id_ = currentThreadId();
   state_->operator_name_ = op.operator_name();
+  state_->op_input_size = op.schema().arguments().size();
+  state_->op_output_size = op.schema().returns().size();
```

Comment:
should we just use state_'s inputs and outputs size instead in num_inputs and num_outputs?

Reply:
For boxed calls with stack, I have encountered more values on the stack than actually pushed/popped by the operator. If it works like an actual call stack, we may have values in the stack from previous calls based on this [note](https://github.com/pytorch/pytorch/wiki/Boxing-and-Unboxing-in-the-PyTorch-Operator-Library). Maybe @smessmer could chime in.

Reply:
In a boxed call, the stack can contain args from calls "further up the stack", not only from the current call.